Documentation: Add VSCode debugger instructions (#2064)
LadybirdBrowser/ladybird: Documentation/EditorConfiguration/VSCodeConfiguration.md

</details>

### Debugging
#### Mac
If you want to run the debugger, first place the content below in `.vscode/launch.json` in the root of the project.

```json 
{
    "version": "0.2.0",
    "configurations": [
        {
            "name": "Attach to WebContent",
            "type": "lldb",
            "request": "attach",
            "program": "${workspaceFolder}/Build/ladybird-debug/bin/Ladybird.app/Contents/MacOS/WebContent",
        }
    ],
}
```

then run Ladybird with the debug preset and with the `--debug-process WebContent` flag:

```bash
CC=$(brew --prefix llvm)/bin/clang CXX=$(brew --prefix llvm)/bin/clang++ BUILD_PRESET=Debug ./Meta/ladybird.sh run ladybird --debug-process WebContent
```

> running Ladybird in this way will pause execution until a debugger is attached which you can run if you go to the Run and Debug menu and run the Attach to WebContent configuration. 

Reviewer: ```suggestion
Running Ladybird in this way will pause execution until a debugger is attached. You can then run the debugger by going to the **Run and Debug** menu and selecting the **Attach to WebContent** configuration. 
```
(just a non-maintainer review suggestion)
